src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_layout.html.twig

```diff
@@ -283,7 +283,7 @@
 
 {% block form_errors -%}
     {%- if errors|length > 0 -%}
-        <span class="{% if form is not rootform %}invalid-feedback d-block{% else %}alert alert-danger{% endif %}">
+        <span class="{% if form is not rootform %}invalid-feedback d-block{% else %}alert alert-danger d-block{% endif %}">
```

Comment:
Let's move the `d-block` class outside the `{% if %}` statement to have it only once in the line